starter-code/css/main.css

```diff
@@ -1,3 +1,46 @@
+/* reset */
+
+/* typography  ...including h1 margins and padding etc */
+
+/* layout */
+    .main {
+        right: 0;
+        left: 0;
+        width: 100%; 
+    }
+
+    .navbar {
+        position: fixed;
+        right: 0;
+        left: 0;
+        top: 0;
+        /* max-width: 100%; */
+        background-color: #FFF;
+        /* border: 0px; */
+        /* display: block; */
+    }
+
+/* components */
+    .nav-buttons {
+        float: right;
+        }
+
+    .nav-buttons ul li{
+        display: inline;
+        /* margins needed */
+    }
+
+/* Where to put this?  */ ul {list-style: none;}
```

Comment:
Try not to style element tags like div or ul. Give them a class or ID instead.

You can put it in components :)